src/containers/Review/ReviewHome.tsx

```diff
@@ -55,10 +66,96 @@ const ReviewHome: React.FC<ReviewHomeProps> = ({ assignments, structure }) => {
                 }}
               />
             ))}
-        </Segment>
-      ))}
+          </Segment>
+        ))}
     </>
   )
 }
 
+type ReviewerAndStageSelectionProps = {
+  filters: Filters | null
+  setFilters: (filters: Filters) => void
+  structure: FullStructure
+  assignments: AssignmentDetailsNEW[]
+}
+
+const ReviewerAndStageSelection: React.FC<ReviewerAndStageSelectionProps> = ({
+  assignments,
+  structure,
+  filters,
+  setFilters,
+}) => {
+  const {
+    userState: { currentUser },
+  } = useUserState()
+
+  useEffect(() => {
+    setFilters({
+      selectedReviewer: currentUser?.userId as number,
+      selectedStage: structure.info.current?.stage.id as number,
+    })
+  }, [])
+
+  const changeFilters = (filterType: keyof Filters) => (_: any, { value }: any) => {
+    if (filters) setFilters({ ...filters, [filterType]: value })
+  }
+
+  if (!filters) return null
+
+  return (
+    <Grid columns="equal">
+      <Grid.Column floated="left">
+        Show tasks for{' '}
+        <Dropdown
+          options={getReviewerOptions(assignments, currentUser?.userId as number)}
+          value={filters?.selectedReviewer}
+          onChange={changeFilters('selectedReviewer')}
+        />
+      </Grid.Column>
+      <Grid.Column floated="right" textAlign="right">
+        STAGE{' '}
```

Comment:
To localise 4

Reply:
75ee2014fa6a0379e4fc44773462dbf921da3088